.ado/templates/apple-nuget-publish.yml

```diff
@@ -54,7 +54,12 @@ steps:
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Release'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig'
+      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides_ios_simulator.xcconfig'
+
+  # TODO: remove soon: Check simulator binary archs
```

Comment:
Do we have to?

Reply:
i am running a manual pipeline build right now. i will remove the block of line 59 after i verified.

Reply:
I meant, I kinda like listing the archs here as a permanent step. I guess we don't _need_ it.

Reply:
i think i rather reduce our time in our nuget builds :)